apps/notifications/src/panel/boot/stylesheets/main.scss

```diff
@@ -284,7 +284,7 @@
 			display: flex;
 			align-items: center;
 			justify-content: center;
-			color: white;
+			color: var(--color-text-inverted);
```

Comment:
I think we may need to just use the hex code here because the studio colors aren't defined in the notifications panel :( 

The reason is that notifications on the front-end of a site (e.g. at foo.wordpress.com) is outside of Calypso. So it would have to include the studio variables differently from how those are included in Calypso. Unfortunately, it doesn't seem like they're defined for notifications, just after double checking in the browser devtools.

I also double checked the other apps (like editing toolkit plugin), and it _does_ look like the variables are defined in the editor on both simple and atomic sites, so the other non-calypso changes should work fine.

Reply:
Ah, thank you for pointing this out! I'll make this change.